lib/fastlane/plugin/sentry/actions/sentry_upload_sourcemap.rb

```diff
@@ -10,14 +10,14 @@ def self.run(params)
         version = "#{params[:app_identifier]}@#{params[:version]}" if params[:app_identifier]
         version = "#{version}+#{params[:build]}" if params[:build]
 
-        sourcemap = params[:sourcemap]
+        sourcemaps = params[:sourcemap]
```

Comment:
When changing the expected type of the config item to `array`, we can do something like this here:

```ruby
sourcemaps = params[:sourcemap]

command = [
  "releases",
  "files",
  version,
  "upload-sourcemaps"
]
command += sourcemaps
```